### DISCUSSION TITLE
code

### DISCUSSION THREAD

**Discussion by garfield-gray** [community/community#10860] [Copilot Conversations] [answered]
mmmm what's the device activation code? :///

**Comment by jansorg** [comment, marked as answer]
It should be shown right inside the IDE or editor :) 
Which editor are you using?

**Reply by garfield-gray** [reply]
Yeah it was written there
I was freaky a little bit
I'm using vim by the way
Thanks

**Reply by TerryPanos** [reply]
Has anyone got the recom3 software so I can activate me recon jet sunglasses please. I just need to get passed the activation screen

**Comment by anhtuansj1269** [comment]
def fahrenheit_to_celsius(fahrenheit):
    """Convert Fahrenheit to Celsius."""
    return (fahrenheit - 32) * 5/9

# Prompt the user for input
fahrenheit = float(input("Enter temperature in Fahrenheit: "))

# Convert and display the result
celsius = fahrenheit_to_celsius(fahrenheit)
print(f"{fahrenheit}°F is equal to {celsius:.2f}°C.")
